### PR TITLE
[ReadTheDocs] Structure and template fixes

### LINKED ISSUE #6209: ReadTheDocs build failure on LORIS API symlinks
**Describe the bug**
ReadTheDocs is failing because it's unable to resolve the following symlinks:

docs/wiki/99\ -\ Developers/LORIS-REST-API-0.0.2.md
docs/wiki/99\ -\ Developers/LORIS-REST-API-0.0.3-dev.md

These were added in https://github.com/aces/Loris/pull/6151

**What did you expect to happen?**
ReadTheDocs and mkdocs should resolve the symlinks and render the documentation.

We may need to modify these to include relative path characters as RTD is served from `docs/` rather than the LORIS root. 

**Additional context**
ReadTheDocs build error:

```
INFO    -  Cleaning site directory 
INFO    -  Building documentation to directory: /home/docs/checkouts/readthedocs.org/user_builds/acesloris/checkouts/latest/_build/html 
INFO    -  The following pages exist in the docs directory, but are not included in the "nav" configuration:
  - React.README.md
  - SQLModelingStandard.md
  - deprecated_wiki/About-superuser.md
  - deprecated_wiki/CentOS-Imaging-installation-transcript.md
  - deprecated_wiki/Code-Customization.md
  - deprecated_wiki/Developer's-Instrument-Guide.md
  - deprecated_wiki/Getting-the-Release.md
  - deprecated_wiki/Guide-to-Loris-React-components.md
  - deprecated_wiki/How-to-Code-an-Instrument.md
  - deprecated_wiki/How-to-make-a-LORIS-module.md
  - deprecated_wiki/Installing-Loris-in-Brief.md
  - deprecated_wiki/Instrument-Groups.md
  - deprecated_wiki/Instrument-Insertion.md
  - deprecated_wiki/Instrument-Scoring.md
  - deprecated_wiki/Instrument-Scripts.md
  - deprecated_wiki/Instrument-Testing-and-Troubleshooting.md
  - deprecated_wiki/LORIS-Dictionary.md
  - deprecated_wiki/LORIS-Form.md
  - deprecated_wiki/LORIS-Module-Testing.md
  - deprecated_wiki/LORIS-Modules.md
  - deprecated_wiki/LORIS-scripts-in-the-tools--directory.md
  - deprecated_wiki/Notification-system.md
  - deprecated_wiki/Open-LORIS.md
  - deprecated_wiki/Other-Imaging-Scripts.md
  - deprecated_wiki/Reloading-MRI-data-for-mislabelled-session.md
  - deprecated_wiki/Updating-your-LORIS.md
  - deprecated_wiki/Upgrading-Loris.md
  - deprecated_wiki/Using-Google-reCAPTCHA.md
  - deprecated_wiki/Working-with-React.md
  - deprecated_wiki/XIN-Rules.md
  - deprecated_wiki/About/About.md
  - deprecated_wiki/Setup/Setup.md
  - deprecated_wiki/Setup/Initial Setup/Backups/Backups.md
  - deprecated_wiki/Setup/Initial Setup/Behavioural Database/Behavioural-Database.md
  - deprecated_wiki/Setup/Initial Setup/Data Querying Tool/Data-Querying-Tool.md
  - deprecated_wiki/Setup/Initial Setup/Enable Mail Server/Enable-mail-server.md
  - deprecated_wiki/Setup/Initial Setup/Imaging Database/Imaging-Database.md
  - deprecated_wiki/Setup/Initial Setup/Install Script/Install-Script.md
  - deprecated_wiki/Setup/Initial Setup/LORIS Modules/Candidate Information Page/Candidate-Information-Page.md
  - deprecated_wiki/Setup/Initial Setup/LORIS Modules/Candidate Parameters/Candidate-Parameters.md
  - deprecated_wiki/Technical/Code-Review-Checklist.md
  - deprecated_wiki/Technical/Developer-Workshop-2015-02-13.md
  - deprecated_wiki/Technical/Developer-Workshops.md
  - deprecated_wiki/Technical/Technical.md
  - wiki/00 - SERVER INSTALL AND CONFIGURATION/README.md
  - wiki/00 - SERVER INSTALL AND CONFIGURATION/01 - LORIS Install/README.md
  - wiki/00 - SERVER INSTALL AND CONFIGURATION/01 - LORIS Install/Macintosh/README.md
  - wiki/01 - STUDY PARAMETERS SETUP/README.md
  - wiki/01 - STUDY PARAMETERS SETUP/01 - Study Variables/README.md
  - wiki/01 - STUDY PARAMETERS SETUP/01 - Study Variables/01 - Identifiers.md
  - wiki/_ARCHIVE/README.md
  - wiki/_DELETED/README.md
  - wiki/_DELETED/API.md
  - wiki/_DELETED/Install-Script-for-16.X.md
  - wiki/_DELETED/Installing-Loris-(After-Installing-Prerequisites).md
  - wiki/_DELETED/LORIS database schema.md
  - wiki/_DELETED/LORIS-Setup-Schematic.md
  - wiki/_DELETED/Request-Accounts-module.md
  - wiki/_DELETED/_Footer.md
  - wiki/_DELETED/_Sidebar.md
  - wiki/_DELETED/Community/Community.md
  - wiki/_DELETED/Community/Development.md
  - wiki/_DELETED/Community/Documentation.md
  - wiki/_DELETED/Community/Get-in-Touch.md
  - wiki/_DELETED/Resources/Instrument Coding Guide/Instrument-Coding-Guide.md
  - wiki/_DELETED/Resources/Ubuntu Upgrading/Upgrading-from-Ubuntu-12.04-to-14.04.md 
WARNING -  Documentation file 'CodingStandards.md' contains a link to 'wiki/99%20-%20Developers/Automated%20Testing.md' which is not found in the documentation files. 
WARNING -  Documentation file 'deprecated_wiki/Setup/Initial Setup/Install Script/Install-Script.md' contains a link to 'deprecated_wiki/Setup/Initial Setup/Install Script/Install-Script-for-16.X' which is not found in the documentation files. 
WARNING -  Documentation file 'wiki/00 - SERVER INSTALL AND CONFIGURATION/01 - LORIS Install/CentOS/README.md' contains a link to '../README.md' which is not found in the documentation files. 
WARNING -  Documentation file 'wiki/00 - SERVER INSTALL AND CONFIGURATION/01 - LORIS Install/CentOS/README.md' contains a link to '../README.md' which is not found in the documentation files. 
WARNING -  Documentation file 'wiki/01 - STUDY PARAMETERS SETUP/01 - Study Variables/03 - Sites.md' contains a link to 'wiki/01 - STUDY PARAMETERS SETUP/01 - Study Variables/SQL Dictionary.md' which is not found in the documentation files. 
WARNING -  Documentation file 'wiki/99 - Developers/Automated Testing.md' contains a link to '../test/UnitTestGuide.md' which is not found in the documentation files. 
ERROR   -  File not found: wiki/99 - Developers/LORIS-REST-API-0.0.2.md 
ERROR   -  Error reading page 'wiki/99 - Developers/LORIS-REST-API-0.0.2.md': [Errno 2] No such file or directory: '/home/docs/checkouts/readthedocs.org/user_builds/acesloris/checkouts/latest/docs/wiki/99 - Developers/LORIS-REST-API-0.0.2.md' 
Traceback (most recent call last):
  File "/home/docs/.pyenv/versions/3.7.3/lib/python3.7/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/docs/.pyenv/versions/3.7.3/lib/python3.7/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/home/docs/checkouts/readthedocs.org/user_builds/acesloris/envs/latest/lib/python3.7/site-packages/mkdocs/__main__.py", line 202, in <module>
    cli()
  File "/home/docs/checkouts/readthedocs.org/user_builds/acesloris/envs/latest/lib/python3.7/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/home/docs/checkouts/readthedocs.org/user_builds/acesloris/envs/latest/lib/python3.7/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/home/docs/checkouts/readthedocs.org/user_builds/acesloris/envs/latest/lib/python3.7/site-packages/click/core.py", line 1259, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/home/docs/checkouts/readthedocs.org/user_builds/acesloris/envs/latest/lib/python3.7/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/docs/checkouts/readthedocs.org/user_builds/acesloris/envs/latest/lib/python3.7/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/home/docs/checkouts/readthedocs.org/user_builds/acesloris/envs/latest/lib/python3.7/site-packages/mkdocs/__main__.py", line 163, in build_command
    ), dirty=not clean)
  File "/home/docs/checkouts/readthedocs.org/user_builds/acesloris/envs/latest/lib/python3.7/site-packages/mkdocs/commands/build.py", line 274, in build
    _populate_page(file.page, config, files, dirty)
  File "/home/docs/checkouts/readthedocs.org/user_builds/acesloris/envs/latest/lib/python3.7/site-packages/mkdocs/commands/build.py", line 170, in _populate_page
    page.read_source(config)
  File "/home/docs/checkouts/readthedocs.org/user_builds/acesloris/envs/latest/lib/python3.7/site-packages/mkdocs/structure/pages.py", line 129, in read_source
    with io.open(self.file.abs_src_path, 'r', encoding='utf-8-sig', errors='strict') as f:
FileNotFoundError: [Errno 2] No such file or directory: '/home/docs/checkouts/readthedocs.org/user_builds/acesloris/checkouts/latest/docs/wiki/99 - Developers/LORIS-REST-API-0.0.2.md'
```

### LINKED ISSUE #6329: Move ReadTheDocs root to be the same as the LORIS root
**Describe the bug**
ReadTheDocs has issues accessing files outside of `docs/` as that is where it is served from. This causes problems if we want to, for example, serve a module README in the wiki, because that file will be under `modules/` which RTD can't access.

**To Reproduce**
e.g. #6209 

**Additional context**
There might be a problem with doing this. I remember when initializing this that there was an issue with `mkdocs.yml` being in the same folder as the RTD root. I think that's why it's in `docs/` in the first place.

If this is the case, then we may have to move documents from other places in the codebase into `docs/` when we desire them to be in the Wiki.

### LINKED ISSUE #6545: [Documentation] ReadTheDocs issue with navigation tree
On [ReadTheDocs](https://acesloris.readthedocs.io/en/latest)

Some section like this one:
[Docs » Study Parameters Setup » Study Variables » Introduction](https://acesloris.readthedocs.io/en/latest/wiki/01%20-%20STUDY%20PARAMETERS%20SETUP/01%20-%20Study%20Variables/00%20-%20Introduction%20to%20Study%20Variables/)

displays the breadcrumb sections out of order:
Docs » Study Variables » Study Parameters Setup » Introduction

cc @johnsaigle

### LINKED ISSUE #6622: [RTD] Page broken on 404 errors
On the ReadTheDocs documentation, the 404 page is broken:
[example](https://acesloris.readthedocs.io/en/latest/fsdf)

### LINKED ISSUE #6623: [ReadTheDocs] Bulleted lists in install documentation do not render properly
Spaces should be added between these lines so that they render in a pretty way.

---
**Screenshots:**

Screen Shot 2020-05-27 at 14 55 20
Screen Shot 2020-05-27 at 14 55 11

### LINKED ISSUE #6736: RTD Build errors
>ERROR   -  Error reading page 'wiki/99 - Developers/LORIS-REST-API-0.0.2.md': [Errno 2] No such file or directory: '/home/docs/checkouts/readthedocs.org/user_builds/acesloris/checkouts/latest/docs/wiki/99 - Developers/LORIS-REST-API-0.0.2.md' 
Traceback (most recent call last):
  File "/home/docs/.pyenv/versions/3.7.3/lib/python3.7/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/home/docs/.pyenv/versions/3.7.3/lib/python3.7/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/home/docs/checkouts/readthedocs.org/user_builds/acesloris/envs/latest/lib/python3.7/site-packages/mkdocs/__main__.py", line 202, in <module>
    cli()
  File "/home/docs/checkouts/readthedocs.org/user_builds/acesloris/envs/latest/lib/python3.7/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/home/docs/checkouts/readthedocs.org/user_builds/acesloris/envs/latest/lib/python3.7/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/home/docs/checkouts/readthedocs.org/user_builds/acesloris/envs/latest/lib/python3.7/site-packages/click/core.py", line 1259, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/home/docs/checkouts/readthedocs.org/user_builds/acesloris/envs/latest/lib/python3.7/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/docs/checkouts/readthedocs.org/user_builds/acesloris/envs/latest/lib/python3.7/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/home/docs/checkouts/readthedocs.org/user_builds/acesloris/envs/latest/lib/python3.7/site-packages/mkdocs/__main__.py", line 163, in build_command
    ), dirty=not clean)
  File "/home/docs/checkouts/readthedocs.org/user_builds/acesloris/envs/latest/lib/python3.7/site-packages/mkdocs/commands/build.py", line 274, in build
    _populate_page(file.page, config, files, dirty)
  File "/home/docs/checkouts/readthedocs.org/user_builds/acesloris/envs/latest/lib/python3.7/site-packages/mkdocs/commands/build.py", line 170, in _populate_page
    page.read_source(config)
  File "/home/docs/checkouts/readthedocs.org/user_builds/acesloris/envs/latest/lib/python3.7/site-packages/mkdocs/structure/pages.py", line 129, in read_source
    with io.open(self.file.abs_src_path, 'r', encoding='utf-8-sig', errors='strict') as f:
FileNotFoundError: [Errno 2] No such file or directory: '/home/docs/checkouts/readthedocs.org/user_builds/acesloris/checkouts/latest/docs/wiki/99 - Developers/LORIS-REST-API-0.0.2.md'

### LINKED ISSUE #6737: Duplicated documentation
There are two folders in `docs/wiki/` since the release: `99_Developers/` and `99 - Developers/`. They probably have mostly similar contents but there is likely synchronization issues because of the decision to change these directories during the release process.

### PR DESCRIPTION
**Pull request by laemtl**
Fixes merge conflicts in the wiki (99 - Developers should not exists).
Fixes several errors including some which prevents the Loris ReadTheDocs to compile 
(last compiled version is 4 months old now).

Demo: https://loris-dev.readthedocs.io/en/latest

Docs is now part of the breadcrumb. I can probably find a way to remove it if that's undesirable.
Symlinks are problematic (and as pointed [here](https://github.com/aces/Loris/pull/6712#issuecomment-642218561), not really useful) as we need (for now) to use relative links across all the documentation to avoid broken links.

* Resolves #6209 
* Resolves #6329
* Resolves #6545
* Resolves #6622
* Resolves #6623
* Resolves #6737
* Resolves #6736

### CODE REVIEW

**Comment by johnsaigle** [comment]
Typically after a release the changes from the release branch will be merged back into master. So for example anything made to `23.0-release` will be merged into `master` after the release come sout. Then we will work for a while, decide on a new release, and create `24.0-release`. When 24 comes out, those changes will go back into `master`, etc.

Usually @ridz1208 is the one who takes care of this.

SO what I'm saying here is that I don't think you'll need to make this PR manually to push changes from 23.0-release to master. Everything will make it there after the release.

Our release process can sometimes be a bit convoluted so let me know if what I said is not clear.

**Comment by johnsaigle** [comment]
Adding needs rebase because of conflicts

File: docs/plugins/mkdocs-root-plugin/mkdocs_root_plugin/plugin.py [hunk 1]
```diff
@@ -0,0 +1,68 @@
+from mkdocs.plugins import BasePlugin
```
Comment (driusan) [inline docs/plugins/mkdocs-root-plugin/mkdocs_root_plugin/plugin.py:1]: What is this script for and why is it in the docs directory?

Comment (laemtl) [reply docs/plugins/mkdocs-root-plugin/mkdocs_root_plugin/plugin.py:1]: This script is a mkdocs plugin which allows .md files outside of the docs folder to be compiled and be part of the readthedocs documentation. Readthedocs has limitations, and it does not support reference to files outside of the doc root (see #6329). This plugin creates a temporary folder and copies all the .md files of the repository there to make links from the wiki to .md in the moodules directory possible.

**Comment by laemtl** [comment]
> The API documentation should not be moved out of the API module.

I'm not sure I understand where is the problem there.

**Comment by driusan** [comment]
The files LorisRESTAPI.md (and the variations of it), and the files that it references (ie. InstrumentFormat).. you moved them out of the API module in this PR.

File: docs/plugins/mkdocs-root-plugin/mkdocs_root_plugin/plugin.py (hunk 1 above)
Comment (driusan) [reply docs/plugins/mkdocs-root-plugin/mkdocs_root_plugin/plugin.py:1]: Does it need to go in the docs directory? What advantage does having a script have over simply moving the doc root to be the same as the LORIS root?

Comment (laemtl) [reply docs/plugins/mkdocs-root-plugin/mkdocs_root_plugin/plugin.py:1]: Readthedocs throw an error if the doc root is changed to be the same as the LORIS root, beacuse this change makes it break. The only way to achieve this is with a plugin.

**Comment by christinerogers** [comment]
Let's discuss this PR in the LORIS meeting next week.  It's stalled, probably needs discussion, and is also important to get merged soon-ish to ensure the documentation for the 23 release (as well as master) and dynamic links ensuring docs across the repo can be rendered properly. 
@driusan I think you're the only maintainer with the credential for our RTD setup on _acesloris_ right now given John's departure.

**Comment by laemtl** [comment]
@driusan Everything has been fixed. I added a note to specify that the actual ReadThedocs is outdated (4month old) as compilation is broken since then.

File: docs/plugins/mkdocs-root-plugin/mkdocs_root_plugin/plugin.py (hunk 1 above)
Comment (laemtl) [reply docs/plugins/mkdocs-root-plugin/mkdocs_root_plugin/plugin.py]: @driusan Scripts have been moved from /docs to /readthedocs.

**Pull request merged**